stubs/resources/views/components/nav-dropdown.blade.php

```diff
@@ -0,0 +1,9 @@
+<li {{ $attributes->merge(['class' => 'dropdown']) }} x-data="{ open: false }" @keyup.escape.window="open = false" @click.away="open = false" @close.stop="open = false">
+    <button class="link" @click="open = ! open" x-bind:aria-expanded="open.toString()">
+        {{ $trigger }}
+    </button>
+
+    <ul role="list" @click="open = false">
```

Comment:
Shouldn't need to add `role="list"` to the `<ul>` and `<ol>`. 

See: [List role best practices](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/list_role#best_practices)

Reply:
This is unfortunately necessary for VoiceOver users— see here: https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html